docs/sphinx/manual/building_hpx.rst

```diff
@@ -521,15 +521,14 @@ libraries, you need to specify them as link dependencies in your build (See
 :ref:`creating_hpx_projects`).
 
 As |hpx| is a modern C++ library, we require a certain minimum set of features
-from the C++11 standard. In addition, we make use of certain C++14 features if
+from the C++17 standard. In addition, we make use of certain C++20 features if
 the used compiler supports them. This means that the |hpx| build system will try
 to determine the highest support C++ standard flavor and check for availability
```

Comment:
My bad. This is outdated. This should say we use C++17 by default, and features from a more recent standard can be opted into explicitly.